### ISSUE TITLE
adb mit OSKR

### ISSUE THREAD

**Issue by moribundant** [digital-dream-labs/lnav-configuration#2]
I have not been able to connect via adb and am wondering if there is a 'catch' that I am missing.

Mac host + OSKR
adbd is running on OSKR at port 5555. Mac has ssh to OSKR, WebViz runs fine, etc. 

**bash-3.2$** adb connect victor:5555
failed to connect to 'victor:5555': Operation timed out

<victor==robot ip; direct address (192.168.x.x) doesn't work either>

**Comment by grant-olson** [comment]
We have been debugging exclusively with ssh. At some point in the past 
Anki stopped using adb so we're not sure how well it works. I was going 
to suggest disabling the firewall but it sounds like you've already done 
that.

**Comment by moribundant** [comment]
Yep, wide open. In fact, the adbd is listening on 5555 and 5037. What it is listening for is anyones guess.
